config/initializers/register_oids.rb

```diff
@@ -1,8 +1,12 @@
 module PostGISTypes
   def initialize_type_map(m = type_map)
     super
-    register_class_with_limit m, 'geometry', ActiveRecord::ConnectionAdapters::PostgreSQLAdapter::OID::SpecializedString
-    register_class_with_limit m, 'geography', ActiveRecord::ConnectionAdapters::PostgreSQLAdapter::OID::SpecializedString
+    %w[
```

Comment:
The way types are registered changed in Rails 7.  We also didn't need a `limit` so we just `register_type` instead of `register_class_with_limit`.

I found this syntax here where they register a bunch of other types (`st_point`, `st_polygon` etc...).  Is there any benefit in registering those as well?
https://github.com/rgeo/activerecord-postgis-adapter/blob/4f4e28fa3c21f4b31b6bdaea484a3872fe3af252/lib/active_record/connection_adapters/postgis_adapter.rb#L67

Reply:
I don't think we have any current plans to use those types directly so I think it is fine to KISS.